lang/es/lang.php

```diff
@@ -8,31 +8,45 @@
         'access_users' => 'Administrar usuarios',
         'access_groups' => 'Administrar grupos',
         'access_settings' => 'Preferencias de usuario',
+        'impersonate_user' => 'Personificar Usuarios'
     ],
     'users' => [
         'menu_label' => 'Usuarios',
         'all_users' => 'Todos los usuarios',
         'new_user' => 'Nuevo usuario',
         'list_title' => 'Administrar usuarios',
-        'activating' => 'Activando...',
         'trashed_hint_title' => 'El usuario tiene su perfil desactivado',
         'trashed_hint_desc' => 'Este usuario tiene desactivado su perfil y  no quiere aparecer en el sitio. Los usuarios pueden reactivar su perfil en cualquier momento iniciando sesión nuevamente.',
         'banned_hint_title' => 'El Usuario ha sido bloqueado',
         'banned_hint_desc' => 'Este usuario ha sido bloqueado por un administrador y no será capaz de iniciar sesión.',
+        'guest_hint_title' => 'Este usuario es un invitado',
+        'guest_hint_desc' => 'El usuario es almacenado sólo para efectos de referencia y necesita registrarse antes de iniciar sesión.',
         'activate_warning_title' => '¡El usuario no se pudo activar!',
         'activate_warning_desc' => 'Este usuario todavía no ha sido activado y no será capaz de iniciar sesión.',
         'activate_confirm' => '¿Realmente desea activar este usuario?',
         'activated_success' => '¡El usuario ha sido activado exitosamente!',
         'activate_manually' => 'Activar usuario manualmente',
+        'convert_guest_confirm' => '¿Convertir este invitado en usuario?',
+        'convert_guest_manually' => 'Convertir a usuario registrado',
+        'convert_guest_success' => 'El usuario ha sido convertido a una cuenta registrada',
+        'impersonate_user' => 'Iniciar sesión como este usuario',
+        'impersonate_confirm' => '¿Estás seguro que quieres iniciar sesión como este usuario? Puede volver a tu estado original cerrando la sesión.',
```

Comment:
Dice `Estás` y luego `Puede`. Creo que lo correcto seria Puedes ya que al inicio empieza informal, que opinas?

Reply:
@noiarek Thanks for your comments. Just so we can track the changes being requested, would you mind translating your request to @bombazama in English as well?

Reply:
Fixed in f9523328ddcded2dc8b7ce675a7b163d6687bbaa
Thanks!